### PR TITLE
Modification of UI and fixed bugs

### PR DESCRIPTION
**Pull request by anjalisoni3655**
- Replaced loading.gif with a linear progress bar in the loading screen(this will also ensure reduced size of apk). Current apk size is 7 MB.
- removed the automatic debug banner mode.
- Fixed bug: the username in the profile page was not displaying when the user was already logged in.
- Added app bar and back arrow button in the contact and profile page.
- Modification in the UI of profile page, contact page, payment history page.
- Progressbar in the payment history page.
Testing: Tested in a local device.
Result: working.
WhatsApp Image 2020-06-21 at 7 24 04 PM

### CODE REVIEW

**Comment by anjalisoni3655** [comment]
@googlebot I consent.

File: home_insurance_app/lib/pages/payment_history.dart [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class TextWidget extends StatelessWidget {
+  TextWidget(
+      {Key key,
```
Comment (khushi2305) [inline home_insurance_app/lib/pages/payment_history.dart:179]: This class is already present . Reuse that

File: home_insurance_app/lib/pages/payment_history.dart [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+                TextWidget(
+                  leftText: 'Cost: ',
+                  rightText: 'Rs. ${_purchase.policy.cost}' ?? '',
+                ),
+                _purchase.offer != null
+                    ? Column(
+                        children: <Widget>[
+                          TextWidget(
+                            leftText: 'Offers Availed: ',
+                            rightText: '${_purchase.offer.requirements}' ?? '',
```
Comment (khushi2305) [inline home_insurance_app/lib/pages/payment_history.dart:159]: mainAxisSize: MainAxisSize.max . Include this to cover the entire row width

File: home_insurance_app/lib/pages/profile.dart [hunk 3]
```diff
@@ -46,6 +56,12 @@ class _ProfileState extends State<Profile> {
               ),
```
Comment (khushi2305) [inline home_insurance_app/lib/pages/profile.dart:56]: Remove the profession since we do not have any info about that

Comment (anjalisoni3655) [reply home_insurance_app/lib/pages/profile.dart:53]: removed the profession from the profile page.

File: home_insurance_app/lib/pages/payment_history.dart (hunk 2 above)
Comment (anjalisoni3655) [reply home_insurance_app/lib/pages/payment_history.dart:161]: covered the entire row width.

File: home_insurance_app/lib/pages/payment_history.dart (hunk 1 above)
Comment (anjalisoni3655) [reply home_insurance_app/lib/pages/payment_history.dart]: reused the textwidget class in both payment page and payment history page. Have put the textwidget class in the common_widget.dart file

**Comment by khushi2305** [comment]
@googlebot I consent.

**Pull request merged**